### DISCUSSION TITLE
editorconfig

### DISCUSSION THREAD

**Discussion by cutbypham** [LazyVim/LazyVim#519] [Show and tell]
> nvim/lua/plugins/util.lua
```lua
return {
  {
     "gpanders/editorconfig.nvim",
     event = "VeryLazy",
   },
}
```

**Comment by folke** [comment]
fyi, when you're using Nightly, you don't need to install editorconfig.nvim.

It will be included in the upcoming Neovim 0.9 release. See http://neovim.io/doc/user/editorconfig.html
